shopinvader_customer_multi_user/services/customer.py

```diff
@@ -22,16 +22,24 @@ def _prepare_params(self, params, mode="create"):
                 vals.update(
                     {
                         "parent_id": company.id,
-                        "type": self.env[
-                            "res.partner"
-                        ]._invader_client_user_type,
+                        "type": vals.get(
```

Comment:
No need to migrate existing partners to the new `type` value?

Reply:
the old value will be removed and replaced w/ the default one "contact".

Reply:
ouch, no, my data was from a manipulated db. The type is gone, indeed. I'll add a mig step.

Reply:
@sebalix fixed, tnx